CHANGELOG.md

```diff
@@ -1,44 +1,23 @@
 # ChangeLog
 
-Version: 1.2.1
+Version: 1.2.2
 
 ### ADDITIONS:
-- Ported decorative blocks from 1.12
-- Added filtering to Output Hatches
-- Added several utility functions to KJS
-- Added more ore generator types (Classic & Cuboid)
-- Added partial NBT support for recipe inputs
-- Added Wood Cutter recipes
+- Ported Powder Barrels, Industrial TNT and Dynamite from 1.12
```

Comment:
Mention the armor?

Reply:
Agreed. #1210 has been reviewed and approved